### DISCUSSION TITLE
/feature/basic/migrate.html

### DISCUSSION THREAD

**Discussion by giscus** [Mereithhh/vanblog-comment#11] [Announcements]
# /feature/basic/migrate.html

VanBlog 的官方网站

https://vanblog.mereith.com/feature/basic/migrate.html

**Comment by 54AllenZhang** [comment]
如何添加只能添加ICP备案，无法添加公安备案

**Reply by Mereithhh** [reply]
自 `v0.42.0` 后就有了，等出现更新提示后更新一下试试吧！
